src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -389,6 +389,23 @@ export function useSwapInputsController({
   const fetchAndUpdateQuote = async (amount: number, isInputAmount: boolean) => {
     if (!assetToSell.value || !assetToBuy.value) return;
 
+    const updateQuoteWithResponse = async (quoteResponse: Quote | CrosschainQuote | QuoteError) => {
+      'worklet';
+      quote.value = quoteResponse;
+
+      if ((quoteResponse as QuoteError)?.error) {
+        inputValues.modify(prev => {
+          return {
+            ...prev,
+            outputAmount: 0,
+            outputNativeValue: 0,
+          };
+        });
```

Comment:
when we receive a quote error, we want to make sure we reset the output values so that they aren't previous quote values